src/GraphQL/ParseGraphQLServer.js

```diff
@@ -52,6 +50,17 @@ class ParseGraphQLServer {
     }
   }
 
+  async _getServer() {
+    const schemaRef = this.parseGraphQLSchema.graphQLSchema;
+    const newSchemaRef = await this.parseGraphQLSchema.load();
+    if (schemaRef === newSchemaRef && this._server) {
+      return this._server;
+    }
+    const options = await this._getGraphQLOptions();
+    this._server = createServer(options);
```

Comment:
I am afraid of initializing the entire server for every schema change. I took a quick look on yoga repo and I saw a lot of things being initialized in here. Isn't there any other way?

Reply:
@davimacedo I added a test to ensure that the server is only recreated only when schema change is detected (should not occur often). Also, the K6 benchmark shows that performance is better.

I discussed with the maintainers of Yoga, that a new option will be added in the future to handle better this use. But for now, it works pretty well. 🙂